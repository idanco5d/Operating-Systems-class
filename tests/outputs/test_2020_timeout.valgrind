==27774== Memcheck, a memory error detector
==27774== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==27774== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==27774== Command: /mnt/c/Users/illus/os/hw/1/smash
==27774== Parent PID: 27757
==27774== 
==27774== 
==27774== FILE DESCRIPTORS: 4 open (3 std) at exit.
==27774== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_2020_timeout.valgrind
==27774==    <inherited from parent>
==27774== 
==27774== 
==27774== HEAP SUMMARY:
==27774==     in use at exit: 0 bytes in 0 blocks
==27774==   total heap usage: 503 allocs, 503 frees, 101,517 bytes allocated
==27774== 
==27774== All heap blocks were freed -- no leaks are possible
==27774== 
==27774== For lists of detected and suppressed errors, rerun with: -s
==27774== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
