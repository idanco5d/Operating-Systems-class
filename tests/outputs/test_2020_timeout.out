smash> smash> smash> smash> smash> timeout_sample_file1.txt
timeout_sample_file2.txt
smash> before
smash> after
smash> 0
end: 2
smash> after
smash> 0
2
smash> smash: got ctrl-C
smash> smash> 0
[1] timeout 3 ../my_sleep 4 & : 9796 1 secs
smash> 2
smash: got ctrl-C
smash> smash> smash> smash> smash> 